common/logging.go

```diff
@@ -242,24 +241,24 @@ func setupFluentD(logOpts map[string]string, debug bool) {
 // setupLogstash sets up and configures Logstash with the provided options in
 // logOpts. If some options are not provided, sensible defaults are used.
 /// TODO fix me later - needs to be tested with a working logstash setup.
```

Comment:
Can we remove it and grab it from git later/track it in a separate issue?

Reply:
What I've started doing is to file a GH issues and refer to it from the comment in the code ie in the form:
```
// GH-1231: text
```